spec/bowling_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'bowling'
+
+describe 'A bowling game' do
+
+  before do
+    @game = Bowling.new
+  end
+
+  it 'can make a new game' do
+    @game = Bowling.new
+  end
+
+  it 'can roll a gutter score' do
+    20.times{@game.roll(0)}
+    expect(@game.score).to eq 0
+  end
+
+  it 'can roll all one scores' do
+    20.times{@game.roll(1)}
+    expect(@game.score).to eq 20
+  end
+
+  it 'can roll a spare' do
+    @game.roll(5)
+    @game.roll(5)
+    @game.roll(3)
+    17.times{@game.roll(0)}
+    expect(@game.score).to eq 16
+  end
+
+  it 'can roll a strike' do
+    @game.roll(10)
+    @game.roll(3)
+    @game.roll(4)
+    16.times{@game.roll(0)}
+    expect(@game.score).to eq 24
+  end
+
+  it 'can roll a perfect game' do
+    12.times{@game.roll(10)}
+    expect(@game.score).to eq 300
+  end
+
+
+end
```

Comment:
Doesn't prevent wrong scores being entered (no limit of 10 per roll)

Reply:
Maybe add an error message when the game is over?  And/or the functionality to output score during the game (atm it outputs an error)

Reply:
And/or a message when the game is over